lib/core/include/irods/library_features.h

```diff
@@ -19,6 +19,11 @@
 ///
 /// \since 4.3.1
 
+/// Defined if the development library supports the C++ ticket administration library.
+///
+/// \since 4.3.1
+#define IRODS_HAS_LIBRARY_TICKET_ADMINISTRATION 202209L
```

Comment:
202209L ?     not 202307?

Reply:
The value represents when the ticket admin lib was first introduced into the dev library.